TestGoModules fails in Go 1.16beta1

```
$ go version
go version go1.16beta1 linux/amd64

$ git log --oneline -1
07afc7d (HEAD -> master, tag: v1.11.0, origin/master, origin/HEAD) update release pipeline slightly (#327)

$ go test ./mage
...
--- FAIL: TestGoModules (0.10s)
    main_test.go:1445: exited with code 1.
        Stdout:
        Stderr: magefile.go:5:8: no required module provides package golang.org/x/text/unicode/norm; try 'go get -d golang.org/x/text/unicode/norm' to add it
        Error: error compiling magefiles
```
Skipped a bunch of debug output.  All other tests pass.  All tests pass with Go 1.15.6.

Can someone reproduce this?

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,6 +9,7 @@ branches:
 # library, I'm not going to worry about older versions for now.
 go:
   - tip
+  - 1.16.x
   - 1.15.x
   - 1.14.x
   - 1.13.x

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,7 +9,7 @@ branches:
 # library, I'm not going to worry about older versions for now.
 go:
   - tip
-  - 1.16.x
+  - 1.16beta1
   - 1.15.x
   - 1.14.x
   - 1.13.x

--- a/mage/main_test.go
+++ b/mage/main_test.go
@@ -1436,6 +1436,18 @@ func Test() {
 	}
 	stderr.Reset()
 	stdout.Reset()
+
+	// we need to run go mod tidy, since go build will no longer auto-add dependencies.
+	cmd = exec.Command("go", "mod", "tidy")
+	cmd.Dir = dir
+	cmd.Env = os.Environ()
+	cmd.Stderr = stderr
+	cmd.Stdout = stdout
+	if err := cmd.Run(); err != nil {
+		t.Fatalf("Error running go mod tidy: %v\nStdout: %s\nStderr: %s", err, stdout, stderr)
+	}
+	stderr.Reset()
+	stdout.Reset()
 	code := Invoke(Invocation{
 		Dir:    dir,
 		Stderr: stderr,

